Gemfile

```diff
@@ -47,10 +47,17 @@ gem "bootsnap", require: false
 
 # Use Active Storage variants [https://guides.rubyonrails.org/active_storage_overview.html#transforming-images]
 # gem "image_processing", "~> 1.2"
+gem "activeadmin"
 
 group :development, :test do
   # See https://guides.rubyonrails.org/debugging_rails_applications.html#debugging-with-the-debug-gem
-  gem "debug", platforms: %i[ mri mingw x64_mingw ]
+  gem "debug", platforms: %i[mri mingw x64_mingw]
+  gem "rspec-rails"
```

Comment:
Setup rspec please (add factory_bot and others utilities config and etc)

Reply:
Add setup for rspec and change factory_bot for factory_bot_rails